dm/ctl/common/config.go

```diff
@@ -139,7 +145,7 @@ func (c *Config) Parse(arguments []string) (finish bool, err error) {
 	}
 
 	if c.MasterAddr == "" {
-		return false, flag.ErrHelp
+		return false, errors.Errorf("--master-addr not provided")
```

Comment:
in addition to this message, maybe we could tell user to add `--help` to show help messages